### ISSUE TITLE
Regression in v3.6.6 : postinstall script in package.json

### ISSUE THREAD

**Issue by debel88** [final-form/react-final-form#347]
### Are you submitting a **bug report** or a **feature request**?

Bug report

### What is the current behavior?

When installing `react-final-form@3.6.6`, I get this error :

```
> react-final-form@3.6.6 postinstall /home/debel/projects-frontend/react-business/node_modules/react-final-form
> node ./scripts/postinstall.js || exit 0

module.js:442
    throw err;
    ^

Error: Cannot find module '/path/to/project/node_modules/react-final-form/scripts/postinstall.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
```

### What is the expected behavior?

Successful package installation

### Sandbox Link

None.

### What's your environment?

react-final-form@3.6.6
node v6.3.1
Linux Mint 17 Cinnamon

### Other information

In the `package.json` of version `3.6.5`, the "postinstall" script was :

```
"postinstall": "opencollective postinstall || exit 0"
```

In version 3.6.6, it has become :

```
"postinstall": "node ./scripts/postinstall.js || exit 0"
```

There is no `scripts` directory published with the package, hence the error.

**Comment by maciejmyslinski** [comment]
Fixed by #345

**Comment by erikras** [comment]
Published fix in [`v3.6.7`](https://github.com/final-form/react-final-form/releases/tag/v3.6.7).
